app/src/App.test.tsx

```diff
@@ -1,8 +1,9 @@
+import React from 'react';
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText("NMIND");
+  const linkElement = screen.getByText(/learn react/i);
```

Comment:
:frowning:

Reply:
Fixed the "test" in https://github.com/nmind/standards-checklist/pull/30/commits/d638a441c666ec5894b4ec57661c54132148ce51